src/animations.js

```diff
@@ -1,3 +1,6 @@
+// Import the karaoke script file before this script is imported.
+/* globals updateKaraoke */
```

Comment:
Extra comment?

Reply:
Line 2 is for ESlint to know that `updateKaraoke` is treated as a global for this file only. I debated putting it in `.eslintrc.js` but wasn't sure if it should be treated as a global across all files since it's only needed in this one. Open to thoughts though :)